Major UI update to 1.5.0

maverickkamal opened the pull request:
# BrowserBuddy Extension Submission

## Project Details

1. **What is the name of your project?**
Visual Timer

2. **How many hours did you spend developing this?**
   _(Give your best estimate)_
26
3. **What problem does your extension solve, or what makes it unique?**
   _(Provide a brief description of the functionality or innovation)_

4. **How many people have tested your extension?**
   _(We recommend at least 3)_

5. **Does your extension work in Chrome and uses Manifest Version 3?**
   _(Your extension must be compatible with Chrome and use Manifest Version 3)_

---


## Additional Comments
I was already approved but I made some major changes including adding additional UI option.
(Feel free to provide any additional context or notes here)

JosiasAurel commented:
please fill the details in the first comment

JosiasAurel reviewed (changes requested):
How's this different from other visual timer extensions?

maverickkamal commented:
What problem does your extension solve, or what makes it unique?
(Provide a brief description of the functionality or innovation)
Visual Timer provides ambient time awareness through subtle color changes in your browser's background - transitioning from blue to red. Unlike traditional timers, it helps you stay time-conscious without breaking focus or switching tabs. Perfect for time-boxing tasks or limiting screen time on specific sites.

How many people have tested your extension?
(We recommend at least 3)
around 11

Does your extension work in both Chrome and Firefox?
(Your extension must be compatible with both browsers)
both chromium and firefox compatible

maverickkamal commented:
But to clarify, I made an update as I was already approved. I added another UI options so thought it will be nice to merge with the other one as I also fixed lots of bugs.

JosiasAurel commented:
> What problem does your extension solve, or what makes it unique? (Provide a brief description of the functionality or innovation) Visual Timer provides ambient time awareness through subtle color changes in your browser's background - transitioning from blue to red. Unlike traditional timers, it helps you stay time-conscious without breaking focus or switching tabs. Perfect for time-boxing tasks or limiting screen time on specific sites.
> 
> How many people have tested your extension? (We recommend at least 3) around 11
> 
> Does your extension work in both Chrome and Firefox? (Your extension must be compatible with both browsers) both chromium and firefox compatible

There's an existing submission that already does this iirc

JosiasAurel commented:
> But to clarify, I made an update as I was already approved. I added another UI options so thought it will be nice to merge with the other one as I also fixed lots of bugs.

alright noted
